### PR TITLE
Add new TdS stage 7 validators

### PR DESCRIPTION
**Pull request by danpaul000**
Add new validators who have joined up to Tour de SOL stage 7 to the Stake-o-matic list.

### CODE REVIEW

File: stake-o-matic/src/validator_list.rs [hunk 1]
```diff
@@ -1,23 +1,30 @@
 solana_sdk::pubkeys!(
     testnet_validators,
     [
+        "121CY7X4snhJcTEJAWG1Q2psae3uSGzXHL",
```
Comment (CriesofCarrots) [inline stake-o-matic/src/validator_list.rs:4]: This pubkey doesn't look right. It only base58 decodes to 25 bytes

Comment (CriesofCarrots) [inline stake-o-matic/src/validator_list.rs:4]: This pubkey doesn't look right. It only base58 decodes to 25 bytes

Comment (CriesofCarrots) [reply stake-o-matic/src/validator_list.rs:4]: Oh hey cool, macro expansion in CI agrees :)

Comment (danpaul000) [reply stake-o-matic/src/validator_list.rs]: Thanks @CriesofCarrots !

**Pull request merged**